data/styles/vehicle/include/Roads.mapcss

```diff
@@ -328,57 +328,73 @@ line|z8[highway=motorway]
 line|z9[highway=trunk],
 line|z9[highway=motorway]
 {width: 1.4;}
+
+line|z10-[highway=trunk],
+line|z10-[highway=motorway]
+line|z10-[highway=motorway_link],
+line|z10-[highway=trunk_link],
+{color: @trunk_orange_medium;}
```

Comment:
*_links had a different color before, why change?

Reply:
No, links started from z13 and had the same _medium_ color:
```
line|z13[highway=trunk],
 line|z13[highway=motorway],
-{color: @trunk_orange_medium; width: 5;}
+{width: 5;}
```

Change because it was a bug before. motorways z10, z12- had color _medium_ and only z11 had _light_ - obvious bug in copy-paste.

Reply:
You're right!
There is the same color bug in the main style also.

Reply:
I'll be great!

Reply:
OMG, looks like the colors in clear style for motorways are opposite to the vehicle one ...
Everywhere is _trunk_orange_light_, but only z10 and z12 are _trunk_orange_medium_.
What should I set? _trunk_orange_light_ everywhere?

Reply:
hmm, highway=primary has the pattern
low zooms : orange
med zooms : orange_medium
high zooms : orange_light

I guess the idea is to have more contrast at low zooms.

So it would make sense to do the same with trunks (start with orange_medium for World, then change to orange_light).

Reply:
```
@trunk: #FF8800;
@trunk_orange: #FF5919;
@trunk_orange_medium: #FF7326;
@trunk_orange_light: #FF7A26;
```
looks like first two are not actually used

Reply:
Seems like the initial idea (a long time ago) was like:
```
line|z4-9[highway=world_towns_level],
line|z4-9[highway=world_level],
{color: @trunk_orange_light;opacity: 1;}
```
But something went wrong and I don't see how these _world_ types are used now.